tests/suites/test_suite_psa_crypto.data

```diff
@@ -6320,6 +6320,10 @@ PSA key derivation: HKDF-Expand SHA-1, request too much capacity
 depends_on:PSA_WANT_ALG_HKDF_EXPAND:PSA_WANT_ALG_SHA_1
 derive_set_capacity:PSA_ALG_HKDF_EXPAND(PSA_ALG_SHA_1):255 * PSA_HASH_LENGTH(PSA_ALG_SHA_1) + 1:PSA_ERROR_INVALID_ARGUMENT
 
+PSA key derivation: TLS 1.2 PSK-to-MS, request too much capacity
+depends_on:PSA_WANT_ALG_SHA_256:PSA_WANT_ALG_TLS12_PSK_TO_MS
+derive_set_capacity:PSA_ALG_TLS12_PSK_TO_MS(PSA_ALG_SHA_256):48U + 1U:PSA_ERROR_INVALID_ARGUMENT
+
```

Comment:
Please add `derive_set_capacity` with the max size for algorithms where the max size can't be tested with `derive_full` because that's too much output.